css/styles.css

```diff
@@ -0,0 +1,122 @@
+/* ====================== Global Styles  ========================== */
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  line-height: 1.5;
+}
+
+html {
+  font-size: 16px;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  font-size: 1em;
+  background-color: #fff;
+}
+
+p {
+  color: #344563;
+}
+
+ul,
+ol {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+.container {
+  width: 90%;
+}
+
+header {
+  display: flex;
```

Comment:
- [ ] As suggested by the reviewer [here](https://github.com/YugynDprodigy10/portfolio/pull/1#discussion_r1149753801), please make sure to follow the CSS best practices and avoid repeating a property 🎯 To learn more about it, please follow [this article](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) 🔗